### PR TITLE
Fix arm64 docker fill-station image build

### LINKED ISSUE #62: Fix arm64 fill-station image
Currently fails with exec /bin/fill_station: no such file or directory

We need to build it with qemu.

Fixed by #67

### PR DESCRIPTION
**Pull request by ZachGarcia42**
Fixes #62

### CODE REVIEW

**Comment by ZachGarcia42** [comment]
* **#67** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/67?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ZachGarcia42 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ZachGarcia42** [comment]
This works when running in Qemu. Will test on pi 4 shortly

**Comment by ZachGarcia42** [comment]
> This works when running in Qemu. Will test on pi 4 shortly

Confirmed working on the Pi!

File: fill/GithubDockerfile [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # Need ubuntu for dynamic linking
+# FROM --platform=linux/arm/v7 arm32v7/ubuntu:22.04 AS run
```
Comment (maxslarsson) [inline fill/GithubDockerfile:2]: Should we remove this comment?

File: fill/GithubDockerfile [hunk 2]
```diff
@@ -10,4 +20,5 @@ RUN ["chmod", "+x", "/bin/fill_station"]
 EXPOSE 50051
 
 # Run the binary
+# ENTRYPOINT [ "tail", "-f", "/dev/null" ]
```
Comment (maxslarsson) [inline fill/GithubDockerfile:23]: Same with this one?

**Review by maxslarsson — CHANGES_REQUESTED** [review]

**Comment by ZachGarcia42** [comment]
Removed extra lines

**Review by maxslarsson — APPROVED** [review]

File: telemetry-proxy/telemetry_proxy
Comment (maxslarsson) [inline telemetry-proxy/telemetry_proxy:1]: Do we want to commit the executable?

**Review by maxslarsson — CHANGES_REQUESTED** [review]

**Comment by ZachGarcia42** [comment]
### Merge activity

* **Sep 30, 1:27 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/67) couldn't merge this PR because it failed for an unknown reason (Changes have been requested by reviewers on this PR).

**Comment by maxslarsson** [comment]
My bad I approved and then I saw that the executable has been commited as well.

File: telemetry-proxy/telemetry_proxy
Comment (ZachGarcia42) [reply telemetry-proxy/telemetry_proxy:1]: No, we do not. This isn't matching what my local instance shows though :(

Comment (ZachGarcia42) [reply telemetry-proxy/telemetry_proxy:1]: Ok, restarting VSCode seemed to make it figure itself out

**Review by maxslarsson — APPROVED** [review]

**Pull request merged**